2-mandatory/7-lane-names.js

```diff
@@ -6,7 +6,11 @@
   HINT: string and array methods that could be helpful (indexOf, filter)
 */
 
-function getLanes() {}
+function getLanes(streetNames) {
+  return streetNames.filter(
+    (streetName) => streetName.indexOf("Lane") === streetName.length - 4
```

Comment:
This works fine, but can you think of a way to do this using one of the methods that we learned this week?